Templates/Templates/ModelMappable.stencil

```diff
@@ -0,0 +1,162 @@
+// https://github.com/SwiftGen/StencilSwiftKit/blob/master/Documentation/tag-set.md
```

Comment:
I understand that you want this for documentation purposes, but that pollutes generated files.

Reply:
I’ll remove the doc links.